src/components/AboutComponent/AboutComponent.tsx

```diff
@@ -1,24 +1,23 @@
 import React from 'react'
 import './AboutComponent.scss'
 import { Helmet } from 'react-helmet'
+import { useTranslation } from "react-i18next";
 
 const About: React.FC = () => {
+
+    const {t} = useTranslation();
+
     return (
         <div className="about">
             <Helmet>
-                <title>CConverter - O projekcie</title>
+                <title>CConverter - About Project</title>
             </Helmet>
 
             <div className="about-layout">
                 <div className="about-layout__item">
                     <div className="about-layout__info">
-                        <p><strong>CConverter</strong> to open-sourcowy projekt stworzony przez <strong>Codrill</strong>.
-                            Mając do dyspozycji 150 walut z
-                            całego świata przeliczenia stają się banalnie proste.</p>
-                        <p>Wystarczy, że wybierzesz waluty które Cię interesują oraz wpiszesz
-                            liczbę. To wszystko. Aplikacja automatycznie przeliczy podaną przez Ciebie kwotę po
-                            odpowiednim i oficjalnym kursie wymiany.
-                        </p>
+                        <p>{t('AboutFirstPhotoFirstParagraph')}</p>
```

Comment:
Using translations this way, we lose HTML markups used inside of content (like 'strong'). Instead of using this classic way of translations, we can try to use 'Trans' component instead, in more complex places like this one. 
Read more about it here:

https://react.i18next.com/latest/trans-component

Reply:
Btw. Can we try to use more meaningful keywords for translations? Idk for example:
1. AboutSectionHowToUse
2. AboutSectionNBP
3. AboutSectionRWD

It will be easier to maintain in the future this way and it's good approach to treat every section like they provide specific type of information.

Reply:
About more meaningful keywords I agree with that. What about specific paragraphs in those section ?

Reply:
Done